docs/2022/reuse/updates/2022-06-02.md

```diff
@@ -103,3 +103,86 @@ GSoC 2022 Discussions - REUSE
 ### Conclusion and Further Plans
 
 - Implement discussed solution in upcoming week.
+
+## Meeting 3
+*(June 9th, 2022)*
+
+GSoC 2022 weekly update
+
+
+### Attendees
+
+- [Anupam Ghosh](https://github.com/ag4ums)
+- [Gaurav Mishra](https://github.com/GMishx)
+- [Shaheem Azmal M MD](https://github.com/shaheemazmalmmd)
+- [Kaushlendra Pratap](https://github.com/Kaushl2208)
+- [Ayush Bhardwaj](https://github.com/hastagAB)
+- [Vivek Kumar](https://github.com/viv9k)
+- [Avinal Kumar](https://github.com/avinal)
+- [Sushant Kumar](https://github.com/its-sushant)
+- [Rohit Pandey](https://github.com/rohitpandey49)
+- [Thanvi Lahari Pendyala](https://github.com/Pendyala-thanvi)
+- [Krishna Mahato](https://github.com/krishna9304)
+- [Soham Banerjee](https://github.com/soham4abc)
+
+### Discussions
+
+- Explored the study material/blog links shared by mentors. 
+- **Can I raise a pull request with a few changes for review?**
+    - Yes. You can raise PR.
+
+## Meeting 4
+*(June 13th, 2022)*
```

Comment:
Hello @rohitpandey49 ,
As from June 13, the coding period started, so please move (this and following) updates to new weekly files under same folder as done in previous run:
https://github.com/fossology/gsoc/tree/main/docs/2021/copyrights/updates

Reply:
Done.